apps/src/code-studio/peer_reviews/PeerReviewSubmissions.jsx

```diff
@@ -1,12 +1,55 @@
 import React from 'react';
-import {Table} from 'react-bootstrap';
+import {Table, FormControl} from 'react-bootstrap';
 import PropTypes from 'prop-types';
+import _ from 'lodash';
 
 class PeerReviewSubmissions extends React.Component {
   static propTypes = {
     submissions: PropTypes.arrayOf(PropTypes.object).isRequired
   }
 
+  state = {}
+
+  componentWillMount() {
+    this.getFilteredResults = _.debounce(this.getFilteredResults, 1000);
+    this.setState({submissions: this.props.submissions});
+  }
+
+  handleTeacherEmailChange = (event) => {
+    this.setState({email_filter: event.target.value});
+
+    // Find something to do email regex
+    this.getFilteredResults();
+  }
+
+  getFilteredResults() {
+    this.loadRequest = $.ajax({
+      method: 'GET',
+      url: `/api/v1/peer_review_submissions/index?filter=escalated&email=${this.state.email_filter}`,
```

Comment:
shouldn't the `filter` param be dynamically set in the request depending on what tab you are on?

Reply:
Ooo good catch. Will fix